LuaMenu/widgets/snd_music_lite.lua

```diff
@@ -54,6 +54,7 @@ local function StartTrack(trackName, volume)
 	end
 	Spring.StopSoundStream()
 	Spring.PlaySoundStream(trackName, volume)
+	Spring.SetSoundStreamVolume(volume)
```

Comment:
what about the line above, doesn't the `volume` parameter to `PlaySoundStream` do anything?

Reply:
It seems to be ignored. I'm guessing this is showing a minor engine bug in music handling.